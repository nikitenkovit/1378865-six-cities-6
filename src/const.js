export const OfferType = {
  APARTMENT: `Apartment`,
  ROOM: `Private Room`,
  HOUSE: `House`,
  HOTEL: `Hotel`
};

export const PathValue = {
  MAIN_SCREEN: `/`,
  FAVORITES_SCREEN: `/favorites`,
  ROOM_SCREEN: `/offer/:id`
};

export const CityCoordinate = {
  AMSTERDAM: {
    latitude: 52.38333,
    longitude: 4.9,
    zoom: 12
  }
}; // скорее всего временно

export const OffersListClassName = {
  CITY_PLACES: `cities__places-list`,
  NEAR_PLACES: `near-places__list`
};

export const BookmarkButtonProperty = {
  ROOM_SCREEN: {
    buttonClass: `property__bookmark-button`,
    iconClass: `property__bookmark-icon`,
    iconWidth: `31`,
    iconHeight: `33`
  },
  OFFER_CARD: {
    buttonClass: `place-card__bookmark-button`,
    iconClass: `place-card__bookmark-icon`,
    iconWidth: `18`,
    iconHeight: `19`
  }
};
