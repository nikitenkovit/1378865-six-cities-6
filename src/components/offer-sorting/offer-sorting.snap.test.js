import React from "react";
import {render} from "@testing-library/react";
import OfferSorting from "./offer-sorting";
import {DEFAULT_SORTING_TYPE} from "../../const";

describe(`OfferSorting tests`, () => {
  it(`Should OfferSorting render correctly`, () => {
    const {container} = render(
        <OfferSorting
          activeType={DEFAULT_SORTING_TYPE}
          onSortChange={jest.fn}
        />);
    expect(container).toMatchSnapshot();
  });
});
