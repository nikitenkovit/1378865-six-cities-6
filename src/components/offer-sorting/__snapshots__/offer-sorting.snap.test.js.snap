// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferSorting tests Should OfferSorting render correctly 1`] = `
<div>
  <form
    action="#"
    class="places__sorting"
    method="get"
  >
    <span
      class="places__sorting-caption"
    >
      Sort by 
    </span>
    <span
      class="places__sorting-type"
      tabindex="0"
    >
      Popular
      <svg
        class="places__sorting-arrow"
        height="4"
        width="7"
      >
        <use
          xlink:href="#icon-arrow-select"
        />
      </svg>
    </span>
    <ul
      class="places__options places__options--custom
    null"
    >
      <li
        class="places__option \\" tabIndex=\\"0\\"
    places__option--active"
      >
        Popular
      </li>
      <li
        class="places__option \\" tabIndex=\\"0\\"
    null"
      >
        Price: low to high
      </li>
      <li
        class="places__option \\" tabIndex=\\"0\\"
    null"
      >
        Price: high to low
      </li>
      <li
        class="places__option \\" tabIndex=\\"0\\"
    null"
      >
        Top rated first
      </li>
    </ul>
  </form>
</div>
`;
